example/urls.py

```diff
@@ -15,6 +15,7 @@
 """
 from django.urls import path, include
```

Comment:
include is already here

Reply:
does not seem to work without importing from django.conf.urls. 
 python --version
Python 3.9.10
django-admin --version
4.1